packages/kotti-ui/source/kotti-breadcrumb/KtBreadcrumb.vue

```diff
@@ -73,12 +83,15 @@ export default defineComponent<KottiBreadcrumb.PropsInternal>({
 
```

Comment:
It is possible to remove this `$darkgray-300`? If yes, you can get rid of the `_variables.scss` import as well here which would be great